noxfile.py

```diff
@@ -146,7 +146,13 @@ def ci_checks_max(session: nox.Session) -> None:
         session: the nox session.
     """
     session.install(
-        ".[docs]", "pylint", "nox", *PYTEST_DEPS, *FMT_DEPS, *DOCSTRING_DEPS, *MYPY_DEPS
+        ".[docs]",
+        "pylint==2.15",
```

Comment:
Just a heads up: in v0.x.x this pinning will have to be moved to `pyproject.toml`.